src/store.ts

```diff
@@ -160,6 +168,9 @@ export interface State {
   tourState?: TourState;
   postConnectTourTrigger?: TourTrigger;
   postImportDialogState: PostImportDialogState;
+
+  predictionInterval: undefined | NodeJS.Timeout | number;
```

Comment:
I've just started using ReturnType<typeof setInterval> so it can't be wrong!

Reply:
[2f738c0](https://github.com/microbit-foundation/ml-trainer/pull/489/commits/2f738c05465c5fa074cad6a28d567309847b3a69)